lib/block-supports/elements.php

```diff
@@ -78,6 +114,8 @@ function () use ( $style ) {
 	return $content;
```

Comment:
This causes the following error for me: `Warning: Undefined variable $content in /var/www/html/wp-content/plugins/gutenberg/lib/block-supports/elements.php on line 114`

Reply:
Thanks, removed!